zerver/webhooks/insping/doc.md

```diff
@@ -0,0 +1,48 @@
+Insping (stylized as !nsping) is a simple uptime and performance monitoring
+tool. You'll get to know when the website or service is up and running or down.
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+{!append-stream-name.md!}
+
+{!append-topic.md!}
+
+## Put incoming webhook to Insping
+
+1. Go to [Insping website](https://www.insping.com). If you haven't created an
+   account, sign up for a free account.
+
+2. Create an organization in the
+   [manage section](https://app.insping.com/#manage)
+   if there isn't one and set an URL that you want to monitor. After it's done,
+   it should look something like this:
+   ![](/static/images/integrations/insping/001.png)
+
+3. Navigate to [the integrations](https://app.insping.com/#integrations) section.
+   This should appear:
+   ![](/static/images/integrations/insping/002.png)
+
+4. Press "Create Webhook", located in the "Webhooks" section. A box will show up:
+   ![](/static/images/integrations/insping/003.png)
+
+5. Recall your incoming webhook URL that you created. Put it in the "Webhook URL"
+   section.
+
+6. Choose an organization that you want to track. the box should look something
+   like this:
+   ![](/static/images/integrations/insping/004.png)
+
+7. After that, click "Create". This should happen:
+   ![](/static/images/integrations/insping/005.png)
+
+8. Finally, to confirm that the incoming webhook works, click the Test button:
+   ![](/static/images/integrations/insping/006.png)
+
+9. A box will show up, asking you to confirm the webhook test. Click "Send Test
+   Data".
+
```

Comment:
Replace the entire point with `To confirm the webhook test, click on **Send Test Data**.` :)

Reply:
Changed into `Click on...`